docs/tutorials/client-setup.md

```diff
@@ -0,0 +1,377 @@
+| "Invalid credentials"                     | Check password and URL format     |
+| "Could not connect"                       | Check if Deluge daemon is running |
+| Special characters in password don't work | URL encode your password          |
+
+### Transmission
+
+Transmission is lightweight and works well with cross-seed.
+
+#### Basic Setup
+
+1. **Enable Remote Access**:
+
+    - Go to Edit > Preferences > Remote
+    - Check "Enable remote access"
+    - Set a username and password
+    - Note the port (default: 9091)
+
+    ![Transmission Remote Access](https://i.imgur.com/example-image-placeholder.jpg)
+
+2. **Configure cross-seed**:
+    ```js
+    module.exports = {
+    	torrentClients: [
+    		"transmission:http://username:password@localhost:9091/transmission/rpc",
+    	],
+    	useClientTorrents: true,
+    };
+    ```
+
+#### Advanced Configuration
+
+1. **Watch Directory Setup**:
+
+    - Consider setting up a watch directory for new torrents
+    - Add a separate watch directory for cross-seed output
+
+2. **Tags/Labels**:
+    - In newer versions of Transmission, you can use tags
+    - Set up tags for identifying cross-seeded content
+
+#### Troubleshooting Transmission
+
+| Problem                             | Solution                                        |
+| ----------------------------------- | ----------------------------------------------- |
+| "401 Unauthorized" error            | Check authentication credentials                |
+| "409 Conflict"                      | Transmission session ID issue; retry connection |
+| Torrents don't appear in cross-seed | Check Transmission RPC path in URL              |
+| Cross-seeds not added               | Verify the download directory permissions       |
+
+### rTorrent / ruTorrent
+
+rTorrent with ruTorrent is powerful but requires more configuration.
+
+#### Basic Setup
+
+1. **Enable XML-RPC**:
+
+    - Edit your .rtorrent.rc file
+    - Ensure the scgi_local or scgi_port setting is configured
+    - Configure your web server (Apache/Nginx) to proxy XML-RPC requests
+
+2. **Configure cross-seed**:
+
+    ```js
+    // For ruTorrent installations:
+    module.exports = {
+    	torrentClients: [
+    		"rtorrent:http://username:password@localhost/rutorrent/plugins/httprpc/action.php",
+    	],
+    	torrentDir: "/path/to/rtorrent/session/directory",
+    };
+
+    // For direct rTorrent RPC:
+    module.exports = {
+    	torrentClients: ["rtorrent:http://username:password@localhost/RPC2"],
+    	torrentDir: "/path/to/rtorrent/session/directory",
+    };
+    ```
+
+#### Advanced Configuration
+
+1. **Session Directory Setup**:
+
+    - Make sure your session directory is set in .rtorrent.rc:
+
+    ```
+    session.path.set = /path/to/session/
+    ```
+
+2. **Watch Directory Configuration**:
+    - Configure watch directories in .rtorrent.rc:
+    ```
+    schedule2 = watch_directory,5,5,"load.start=/path/to/watch/*.torrent"
+    ```
```

Comment:
cross-seed uses direct client injection for everything. you should not have to set up any watch directories for anything related to cross-seed.